winpty/src/windows.rs

```diff
@@ -19,6 +20,7 @@ pub enum ErrorCodes {
     AgentDied,
     AgentTimeout,
     AgentCreationFailed,
+    UnknownError(u32)
```

Comment:
```suggestion
    UnknownError(u32),
```

Did you run `rustfmt` on this? I thought trailing commas are added automagically.